cmd/minishift/cmd/start_preflight.go

```diff
@@ -534,8 +534,8 @@ func validateOpenshiftVersion() bool {
 	return true
 }
 
-// checkOcTagOnGithub return true if specified version of OpenShift released.
-func checkOcTagOnGithub() bool {
+// checkOpenShiftRelease return true if specified version of OpenShift released
+func checkOpenShiftRelease() bool {
```

Comment:
@budhrg @LalatenduMohanty  What are you planning to do here? Do you folks going to overwrite this function with downstream bits?

Reply:
Yes @praveenkumar.

Or create a new function downstream and use it instead of modify above. 

I think the latter approach would be nice since we would have one line of modification which might have potential merge conflict.

Reply:
@budhrg Right that what I suggest to have a new function downstream instead modify above and call that in the `preflightCheckSucceedsOrFails` then I don't think modifying the name of this function is required.

Reply:
Sure. Make sense

Reply:
Having a separate function for OCP oc binary makes sense. However I think the function name should be `checkOriginReleaseTag`. Also we should be checking for releases not tags in Origin repository so the function name should be `checkOriginRelease` and the downstream equivalent should be `checkOcpRelease`

Reply:
> Also we should be checking for releases not tags in Origin repository so the function name should be checkOriginRelease and the downstream equivalent should be checkOcpRelease

@LalatenduMohanty this is not the scope of the issue. We can address it in follow up issue.

Reply:
@budhrg  It is not much work actually. But if you do not want to take it up in this issue thats fine too.